dev/io.openliberty.microprofile.telemetry.2.0.logging.internal.container_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal/container/fat/JULDuplicateTest.java

```diff
@@ -0,0 +1,136 @@
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+
+@RunWith(FATRunner.class)
+public class JULDuplicateTest {
+
+    private static Class<?> c = JULDuplicateTest.class;
+
+    @Server("TelemetryAgentJULMessages")
+    public static LibertyServer server;
+
+    public static final String SERVER_XML_ALL_SOURCES = "allJULSources.xml";
+
+    private static final String[] EXPECTED_FAILURES = { "CWMOT5005W", "SRVE0315E", "SRVE0777E" };
+
+    @ClassRule
+    public static GenericContainer<?> container = new GenericContainer<>(new ImageFromDockerfile()
+                    .withDockerfileFromBuilder(builder -> builder.from(TestUtils.IMAGE_NAME)
+                                    .copy("/etc/otelcol-contrib/config.yaml", "/etc/otelcol-contrib/config.yaml"))
+                    .withFileFromFile("/etc/otelcol-contrib/config.yaml", new File(TestUtils.PATH_TO_AUTOFVT_TESTFILES + "config.yaml"), 0644))
+                    .withLogConsumer(new SimpleLogConsumer(JULDuplicateTest.class, "opentelemetry-collector-contrib"))
+                    .withExposedPorts(4317, 4318);
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        TestUtils.trustAll();
+        WebArchive telemetryLogApp = ShrinkWrap
+                        .create(WebArchive.class, "MpTelemetryLogApp.war")
+                        .addPackage(
+                                    "io.openliberty.microprofile.telemetry.logging.internal.container.fat.MpTelemetryLogApp")
+                        .addAsManifestResource(new File("publish/resources/META-INF/microprofile-config.properties"),
+                                               "microprofile-config.properties");
+
+        ShrinkHelper.exportDropinAppToServer(server, telemetryLogApp,
+                                             DeployOptions.SERVER_ONLY);
+
+        server.addEnvVar("OTEL_EXPORTER_OTLP_ENDPOINT", "http://" + container.getHost() + ":" + container.getMappedPort(4318));
+
+        server.copyFileToLibertyServerRoot("agent-260/opentelemetry-javaagent.jar");
+        server.startServer();
+
+        //Read to run a smarter planet
+        server.waitForStringInLogUsingMark("CWWKF0011I");
+        server.setMarkToEndOfLog();
+    }
+
+    /**
+     * Ensure JUL message logs are not duplicated when the OpenTelemetry Java agent is active with mpTelemetry-2.0.
+     */
+    @Test
+    public void testNoDuplicateJULMessageLogsWithOpenTelemetryAgent() throws Exception {
+        // Wait for the JUL message (SRVE0250I) to appear in the logs.
+        server.waitForStringInLog("SRVE0250I");
+
+        // Wait for the second message (SESN0176I) to arrive, to ensure all the occurrences of the first message (SRVE0250I) has arrived.
+        server.waitForStringInLog("SESN0176I");
+
+        RemoteFile messageLogFile = server.getDefaultLogFile();
+        setConfig(SERVER_XML_ALL_SOURCES, messageLogFile, server);
+
```

Comment:
should add the setConfig to the BeforeClass, so we ensure everything is configured before we start the test.